### PR TITLE
676 script: remove refs from workflows

### PR DESCRIPTION
**Pull request by turtle321**

### CODE REVIEW

File: scripts/workflows/remove_refs.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+def remove_references(workflow_id):
+    wf = workflow_object_class.get(workflow_id)
+    print 'Workflow {} is currently in position {}'.format(workflow_id,
+                                                           wf.callback_pos)
+    # Note that an empty list is not schema compliant.
+    del wf.data['references']
+    wf.save()
+    db.session.commit()
+
+    res = wf.continue_workflow(delayed=True)
```
Comment (ammirate) [inline scripts/workflows/remove_refs.py]: If we don't put the `continue_next` param, you are going to have a robot upload error after, right?

Comment (ammirate) [reply scripts/workflows/remove_refs.py]: Check this guy https://github.com/inveniosoftware-contrib/invenio-workflows/blob/master/invenio_workflows/api.py#L324

Comment (turtle321) [reply scripts/workflows/remove_refs.py]: "continue_next" seems to be the default, see: https://github.com/inveniosoftware-contrib/invenio-workflows/blob/master/invenio_workflows/api.py#L346
Anyway I will add it to be more explicit

**Comment by ammirate** [comment]
So, after discussing IRL, `restart_task` is the right thing to do for continuing the workflow :+1:

**Pull request merged**